src/theme/components/card.ts

```diff
@@ -91,9 +97,9 @@ const Card = {
 
 const CardActions = {
   baseStyle: (props) => {
-    const { bottomBorder, center, layout, topBorder } = props;
+    const { bottomBorder, isCentered, layout, topBorder } = props;
     let justifyContent = null;
-    if (center) {
+    if (isCentered) {
       justifyContent = "center";
```

Comment:
I just realized this is a bug.  The content of `CardActions` should only be certered hotizontally when `isCentered` is applied to `CardLayouts.Column`.  For `Cardlayouts.Row`, the content in `CardActions` should remain left justified.